app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
Toán tử x ||= "something" là viết tắt của x = x || "something". Nó trả về x nếu x khác nil, ngược lại trả về "something" ạ